main.go

```diff
@@ -24,19 +24,23 @@ var (
 )
 
 func main() {
-	err := mainError()
+	err := mainError(context.Background())
 	if err != nil {
 		panic(fmt.Sprintf("%#v", err))
 	}
 }
 
-func mainError() error {
+func mainError(ctx context.Context) error {
```

Comment:
```suggestion
func mainE(ctx context.Context) error {
```

According to https://github.com/giantswarm/fmt/blob/master/go/panic.md.